core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -545,6 +547,12 @@ public void onClosed(Channel c, IOException cause) {
 
         String slaveVersion = channel.call(new SlaveVersion());
         log.println("Remoting version: " + slaveVersion);
+        VersionNumber agentVersion = new VersionNumber(slaveVersion);
```

Comment:
Not sure whether we still care, but I doubt the fallback `< 1.335` will make a useful version number.

Reply:
I don't